src/app/body/board/sprint-details/sprint-details.component.ts

```diff
@@ -23,6 +23,7 @@ export class SprintDetailsComponent implements OnInit {
   componentName :string="SPRINT-DETAILS"
   filterSprintNumber: number;
   sprintStatus: string;
+  enableLoader: boolean = false;
```

Comment:
Loader is being implemented but not available in html, call the loading spinner in css until the api call is completed.